src/components/pages/templates-manage/TemplatesManage.js

```diff
@@ -40,13 +41,16 @@ class TemplatesManage extends React.Component {
   }
 
   render() {
-    const { areasOptions, languageOptions } = this.props;
+    const { areasOptions, languageOptions, loading, mode } = this.props;
     return (
       <div>
         <Hero
-          title="templates.create"
+          title={mode === 'manage' ? "template.manage" : "templates.create"}
```

Comment:
The status are better in caps. Also its better to use booleans if you don't have three states so you don't have to care about typos. Not sure if you are using it for dynamic purposes